Sysdig Secure Source (c2c Collector) (#5037)
SumoLogic/sumologic-documentation: blog-service/2025-02-17-collection.md

---
title: Sysdig Secure Source (Collection)
image: https://help.sumologic.com/img/sumo-square.png
keywords:
  - apps
  - sysdig-secure-source
hide_table_of_contents: true    
---

import useBaseUrl from '@docusaurus/useBaseUrl';

<a href="https://help.sumologic.com/release-notes-service/rss.xml"><img src={useBaseUrl('img/release-notes/rss-orange2.png')} alt="icon" width="50"/></a>

> We're excited to announce the release of our new cloud-to-cloud source for Sysdig Secure. This source helps you collect the complete scan results produced by the scanner from Sysdig API and send them to Sumologic for streamlined analysis. [Learn more](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sysdig-secure-source).

Reviewer: ```suggestion
We're excited to announce the release of our new cloud-to-cloud source for Sysdig Secure. This source aims to collect the scan results from the scanner using Sysdig API and send them to Sumologic.and send them to Sumologic for streamlined analysis. [Learn more](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sysdig-secure-source).
```
